### PR TITLE
Frontend calendar events

### PR DESCRIPTION
**Pull request by beltrancaliz**

### CODE REVIEW

File: frontend/src/app/stocks/containers/Stocks.js [hunk 1]
```diff
@@ -61,6 +63,37 @@ export default class Stocks extends React.Component {
     }
 
 
+
+  }
+
+  buyStock(stock, quantity, price){
```
Comment (hgzimmerman) [inline frontend/src/app/stocks/containers/Stocks.js]: I think this has been supplanted by another function.

File: frontend/src/app/home/containers/Home.js [hunk 2]
```diff
@@ -121,11 +121,66 @@ export default class Home extends React.Component {
...
+      alert("Can't sell more shares than you own!")
+    }
+    else {
+      alert(`Sold ${quantity}x ${stock} shares!`)
+    }
+
+  }
+
+
+  postFeed() {
```
Comment (hgzimmerman) [inline frontend/src/app/home/containers/Home.js]: I think this has been supplanted by another function.

File: frontend/src/app/home/containers/Home.js [hunk 3]
```diff
@@ -121,11 +121,66 @@ export default class Home extends React.Component {
           });
           return null;
         });
+
+
+  }
+
+  sellStock(stock, quantity, price){
```
Comment (hgzimmerman) [inline frontend/src/app/home/containers/Home.js]: This function doesn't do anything and can go away.

File: frontend/src/config/auth.js [hunk 4]
```diff
@@ -3,6 +3,9 @@ import MenuItem from '@material-ui/core/MenuItem';
 import { Route, Redirect, withRouter } from 'react-router-dom';
 
 export const PrivateRoute = function PrivateRoute({ component: Component, ...rest }) {
+  if (!fakeAuth.isAuthenticated){
```
Comment (hgzimmerman) [inline frontend/src/config/auth.js]: FakeAuth has been removed. These lines should be removed.

File: frontend/src/app/home/containers/Home.js (hunk 3 above)
Comment (beltrancaliz) [reply frontend/src/app/home/containers/Home.js]: Will remove

File: frontend/src/app/home/containers/Home.js (hunk 2 above)
Comment (beltrancaliz) [reply frontend/src/app/home/containers/Home.js]: Will remove

**Pull request merged**